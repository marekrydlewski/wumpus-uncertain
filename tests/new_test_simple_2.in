4 4
0.8
????
????
?O??
O??B
