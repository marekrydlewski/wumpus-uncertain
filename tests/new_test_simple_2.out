0.80 0.80 0.80 0.80 
0.80 0.00 0.80 0.80 
0.00 0.00 0.00 0.83 
0.00 0.00 0.83 0.00 
